frontend/packages/dev-console/integration-tests/tests/pipeline.scenario.ts

```diff
@@ -112,12 +132,71 @@ describe('Pipeline', async () => {
       `oc create -f ./packages/dev-console/integration-tests/views/simple-pipeline-demo.yaml -n ${testName}`,
     );
     await pipelinecheckStatus();
-    await browser.wait(until.visibilityOf(pipelineTable));
-    expect(pipelineTableBody.element(by.css('[data-test-id="simple-pipeline"]')).isPresent()).toBe(
-      true,
+    await browser.wait(until.visibilityOf(pipelineTableBodyElement(testName, 'simple-pipeline')));
+    expect(element(by.css('[data-test-id="simple-pipeline"]')).isPresent()).toBe(true);
+    await browser.wait(until.visibilityOf(pipelineTableBodyElement(testName, 'new-pipeline')));
+    expect(element(by.css('[data-test-id="new-pipeline"]')).isPresent()).toBe(true);
+  });
+
+  it('run pipeline and generate pipeline logs scenario', async () => {
+    await switchPerspective(Perspective.Developer);
+    await browser.wait(until.visibilityOf(pageSidebar));
+    await pipelinecheckStatus();
+    await execSync(
+      `oc create -f ./packages/dev-console/integration-tests/views/sample-app-pipeline.yaml -n ${testName}`,
     );
-    expect(pipelineTableBody.element(by.css('[data-test-id="new-pipeline"]')).isPresent()).toBe(
-      true,
+    await browser.wait(until.visibilityOf(pipelinetableParent));
+    await browser.wait(until.presenceOf(samplePipelineElement));
+    await samplePipelineElement.getText().then(function(divText) {
+      expect(divText).toBe('sample-app-pipeline');
+      expect(samplePipelineElement.isPresent()).toBe(true);
+    });
```

Comment:
Why not continue to use await instead of jumping between the two implementations.